packages/server/src/ratings.ts

```diff
@@ -0,0 +1,75 @@
+import {
+    GameResponse,
+    AbstractGame,
+    UserRanking,
+  } from "@ogfcommunity/variants-shared";
+  import { Glicko2, Player } from "glicko2";
+  import { updateUserRating, getUser } from "./users";
+  
+  const ranking = new Glicko2({
+    tau: 0.5,
+    rating: 1500,
+    rd: 350,
+    vol: 0.06,
+  });
+  
+  const matches: [Player, Player, number][] = [];
+  
+  export async function updateRatings(
+    game: GameResponse,
+    game_obj: AbstractGame<object, object>,
+  ) {
+    // Rating system currently only for two player games
```

Comment:
I notice this function has a lot of duplication (do X for player 0, do X for player 1).  It would be better to split it out into component functions like `getGlickoPlayer(id)` and `userRatingFromGlickoPlayer(player)`.  This may make it easier to unit test as well.

Reply:
I would agree!